app/views/idv/finance/new.html.slim

```diff
@@ -13,3 +13,5 @@ p.italic Please provide only one of the following:
   .mt4.mb1
     = link_to 'Cancel', idv_cancel_path, class: 'btn btn-primary bg-gray mr2'
     = f.button :submit, 'Continue', class: 'btn btn-primary'
+
+= page_before_unload_js
```

Comment:
It's not clear to me how to test this since even a phantomjs feature test has no concept of a "window" afaik.